doc/index.rst

```diff
@@ -50,13 +61,10 @@ materials and other geometry-related information.
 ..  _CNERG: https://cnerg.github.io
 ..  _MOAB: https://press3.mcs.anl.gov/sigma/moab-library
```

Comment:
I apologize for any inconvenience caused, but it seems that the previous link is no longer active. Please find the updated link provided below.

```suggestion
..  _MOAB: https://sigma.mcs.anl.gov/moab-library/
```